lib/hanami/configuration.rb

```diff
@@ -42,7 +43,7 @@ def initialize(&blk)
     #     # ...
     #   end
     def mount(app, options)
-      mounted[app] = App.new(app, options.fetch(:at))
+      mounted[app] = App.new(app, options.fetch(:at), options.fetch(:host) { nil })
```

Comment:
Can we use `options[:host]` ?
Should we add this option to documentation and add example?